guide/14-deep-learning/how_deeplabv3_works.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<p>research paper citations should follow APA style </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Done</p>